docs/HowToDevelopLoadableApps.md

```diff
@@ -0,0 +1,216 @@
+# How To Support Loadable Applications on TizenRT
+
+## CONTENTS
+
+- [Overview of Loadable Apps](#overview-of-loadable-apps)
+- [Assumptions & Limitations](#assumptions-&-limitations)
+- [Enable the existing loadable apps](#enable-the-existing-loadable-apps)
+- [Add new loadable apps configuration](#Add-new-loadable-apps-configuration)
+
+## Overview of Loadable Apps
+- When protected build (CONFIG_BUILD_PROTECTED) is selected, then a separate binary is generated for each user space application apart from a separate kernel binary.
+- There is an MPU based memory isolation & protection in place for each application. These apps are called loadable as they can be
+reloaded into their memory space even after fault detection.
+- TizenRT provides 2 sample loadable apps called micom and wifi in TizenRT 3.0.
+- In this example, micom is a critical, high priority app and wifi is non-critical low priority app.
```

Comment:
Could you explain the priority policy for Real-time Threads, Non-real-time Threads and Binary Manager, respectively?

Reply:
done